plugins/user-settings/src/index.ts

```diff
@@ -16,3 +16,4 @@
 export { plugin } from './plugin';
 export { Settings } from './components/Settings';
 export { SettingsPage as Router } from './components/SettingsPage';
+export * from './components/AuthProviders';
```

Comment:
If you're in the mood it'd be nice to follow https://backstage.io/docs/architecture-decisions/adrs-adr004, but I see a lot of the existing code does not 😅

Reply:
I can do that and will try to fix at least the existing imports in this file too...
You mean to introduce and index.ts in /components right?

Reply:
Done

Reply:
He's off call today, but yeah that's the essence of it

Reply:
If you would like to do it in a separate PR I'm happy to merge too.

Reply:
I forgot to commit the new index file -.-